bundles/org.eclipse.equinox.http.jetty/META-INF/MANIFEST.MF

```diff
@@ -9,14 +9,11 @@ Bundle-Activator: org.eclipse.equinox.http.jetty.internal.Activator
 Import-Package: javax.servlet;version="[3.1.0,5.0.0)",
  javax.servlet.http;version="[3.1.0,5.0.0)",
  org.eclipse.equinox.http.servlet;version="[1.2.0,2.0.0)",
+ org.eclipse.jetty.ee8.nested;version="[12.0.1,13.0.0)",
+ org.eclipse.jetty.ee8.servlet;version="[12.0.1,13.0.0)",
  org.eclipse.jetty.http;version="[12.0.1,13.0.0)",
  org.eclipse.jetty.server;version="[12.0.1,13.0.0)",
- org.eclipse.jetty.server.handler;version="[12.0.1,13.0.0)",
  org.eclipse.jetty.session;version="[12.0.1,13.0.0)",
- org.eclipse.jetty.ee8.servlet;version="[12.0.1,13.0.0)",
- org.eclipse.jetty.ee8.nested;version="[12.0.1,13.0.0)",
- org.eclipse.jetty.util;version="[12.0.1,13.0.0)",
```

Comment:
This needs to be fixed in PDE (or bnd...) as it leads to 

> Access restriction: The method 'AbstractLifeCycle.stop()' is not API (restriction on classpath entry '/home/jenkins/agent/workspace/equinox_PR-1080/.m2/repository/org/eclipse/jetty/jetty-util/12.0.23/jetty-util-12.0.23.jar')

I have not yet found out why bndlib thing the package is not referenced, maybe because the method is overriden in a subclass and the compiler emits a reference to the subclass.